teamcity/tests/docker/docker-compose.yaml

```diff
@@ -1,23 +1,25 @@
 version: '3'
-
 # Adapted from https://github.com/JetBrains/teamcity-docker-samples
 services:
   teamcity-server:
     container_name: teamcity-server
     image: jetbrains/teamcity-server:latest
+    platform: linux/amd64
     ports:
       - "8111:8111"
 
   teamcity-agent:
     container_name: teamcity-agent
     image: jetbrains/teamcity-agent:latest
+    platform: linux/amd64
     environment:
       - AGENT_NAME=regular_agent
       - SERVER_URL=http://teamcity-server:8111
 
   teamcity-minimal-agent:
     container_name: teamcity-minimal-agent
```

Comment:
Where can I learn more about the difference between regular and minimal agents?

Reply:
* https://hub.docker.com/r/jetbrains/teamcity-agent/
* https://hub.docker.com/r/jetbrains/teamcity-minimal-agent/

> This [regular agent] image adds a TeamCity agent suitable for Java development

Reply:
Oh, the "agent" here isn't a datadog agent 🤣